plugins/modules/azure_rm_virtualmachine.py

```diff
@@ -2283,6 +2296,7 @@ def exec_module(self, **kwargs):
                         vm_resource.license_type = self.license_type
 
                     if self.vm_identity is not None:
+                        current_user_assigned_identities_dict = {}
```

Comment:
I noticed a linting error where this variable could possibly be referenced without having been defined.